homeassistant/components/dsmr_reader/definitions.py

```diff
@@ -215,7 +186,6 @@ def tariff_transform(value):
     },
     "dsmr/day-consumption/gas": {
         "name": "Gas usage",
-        "icon": "mdi:counter",
         "device_class": VOLUME_CUBIC_METERS,
```

Comment:
This is not a valid device class. Here are the sensor device classes:
https://github.com/home-assistant/core/blob/b790c2e79d463ee016a3e02a01d74e3d4f2d152b/homeassistant/components/sensor/__init__.py#L37-L50